Lib/urllib/parse.py

```diff
@@ -38,29 +39,37 @@
            "DefragResult", "ParseResult", "SplitResult",
            "DefragResultBytes", "ParseResultBytes", "SplitResultBytes"]
 
-# A classification of schemes ('' means apply by default)
-uses_relative = ['ftp', 'http', 'gopher', 'nntp', 'imap',
+# A classification of schemes.
+# We have '' as scheme value to accommodate for the default value of scheme
+# arg in urlsplit and urlparse.
```

Comment:
Better, but without having used argparse yet, I think something like "Use ' ' as the *scheme* value to get the default scheme in urlsplit and urlparse." might be even better.